KsApi/models/ProjectTests.swift

```diff
@@ -213,6 +213,14 @@ final class ProjectTests: XCTestCase {
     XCTAssertEqual(2_000, project.stats.pledgedUsd)
   }
 
+  func testGoalCurrentCurrency() {
```

Comment:
I'm of the opinion that we can make this function test a value that has precision, so we Cana true test of the function and know it actually a `rounded` value.